github/GithubObject.py

```diff
@@ -170,7 +170,7 @@ def parseDatetime(s):
                 # The Downloads API has been removed. I'm keeping this branch because I have no mean
                 # to check if it's really useless now.
                 return datetime.datetime.strptime(s, "%Y-%m-%dT%H:%M:%S.000Z")  # pragma no cover (This branch was used only when creating a download)
-            elif len(s) == 25:
+            elif len(s) >= 25:
```

Comment:
What's the rationale for this?

Reply:
In fact, I am not familiar with the checks here, is it because the length of the datetime string could vary, for some APIs?

Reply:
I  modified this check because the migration object contains a date with timezone `2015-07-06T15:33:38-07:00` but couldn't find out a way to format it. So I now this ignores the timezone information and simple parse the date into `datetime` object.
P.S. - If you have something on (how to parse the date) please let me know.

Reply:
Okay, I'll do it

Reply:
I mean can you parse the string without changing "==" to ">="

Reply:
@singh811 I really want to move this forward, can you revert to my last question? Also there seems to be some conflicts from master that needs to be resolved.

Reply:
Since the date directly comes from the api there is no way to parse that date without using >= condition.

Reply:
[Here](https://developer.github.com/v3/migrations/orgs/#get-the-status-of-an-organization-migration)

Reply:
Which one exactly? The max length I can see is 25.